content/docs/gke/deploy/monitor-iap-setup.md

```diff
@@ -242,3 +245,14 @@ problems:
     and add the redirect URI listed in the error message to the list of 
     authorized URIs. For more information, read the guide to 
     [setting up OAuth for Cloud IAP](/docs/gke/deploy/oauth-setup/).
+
+## Expiry of the SSL certificate from Let's Encrypt
+
+Let's Encrypt recommends renewing your certificate when the
```

Comment:
As worded it sounds like users need to take action to renew their certificates. In practice no action should be needed because we run an agent in the cluster to renew the certificate automatically.

Reply:
Ah thanks, I didn't know that. (I occasionally get emails from Let's Encrypt reminding me that my cert is about to expire.)

Fixed. PTAL.
https://deploy-preview-796--competent-brattain-de2d6d.netlify.com/docs/gke/deploy/monitor-iap-setup/#expiry-of-the-ssl-certificate-from-let-s-encrypt